src/pages/community/patterns/create-flows/index.mdx

```diff
@@ -3,367 +3,440 @@ title: Create flows
 description:
   To create is to generate a new resource. Creating is distinct from adding, but
   the two are often confused. To add a resource is to include a resource that
-  already exists.
+  already exists. This pattern focuses on how to place the user in a flow when
+  there's a need for creating a new resource.
 ---
 
 #### Maintainers:
 
-[Vikki Paterson](https://github.com/vikkipaterson),
-[Chloe Poulter](https://github.com/chloepoulterdesign)
+[Andrew Smith](https://w3.ibm.com/bluepages/profile.html?uid=4G8681897), 
+[Andrea Garcia](https://w3.ibm.com/bluepages/profile.html?uid=4G0746897)
 
-<PageDescription>
+## Overview
+
+Users often need to create different types of assets - they can have simple or
+complex requirements. The Create pattern offers simple (single step) and complex
+(multistep) variations in one of the following formats: Modal, Side panel,
+Narrow tearsheet, Wide tearsheet, and Full-page.
 
 To create is to generate a new resource. Creating is distinct from adding, but
 the two are often confused. To add a resource is to include a resource that
-already exists.
+already exists. Editing a resource is also different from creating a new one.
+This pattern focuses on how to place the user in a flow when there's a need for
+creating a new resource.
 
-</PageDescription>
+Do not use this pattern for adding or editing a resource.
```

Comment:
I would link to the bottom "other use cases" section here for more clarification.

Reply:
done :)